vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/model/StepDefinition.java

```diff
@@ -32,6 +32,8 @@
     private final StepType stepType;
 
     private boolean deprecated;
+    private boolean composite;
+    private boolean dynamic;
```

Comment:
no hashCode/equals updates?

Reply:
I think that module where the step is placed, step as string and its docs is enough to determine equality